### ISSUE TITLE
VoiceOver users cannot compose a message

### ISSUE THREAD

**Issue by sempruijs** [vector-im/element-x-ios#1488]
**Is your feature request related to a problem? Please describe.**

Absolutely. VoiceOver users dubbel tap on a textfield to show the keyboard. 
But that is not the case with the "compose a message" textfield. This is so important.
Visually impaired users cannot write a message to someone.

**Describe the solution you'd like**

When you double tap (activate) the textfield with VoiceOver, the keyboard should pop up so Visually Impaired users can compose a message.

**Describe alternatives you've considered**

There are no alternatives. This is what VoiceOver users expect when they activate a textfield.

**Additional context**

I could only reproduce this problem on a real device, not a simulator.
Element X is already in the App Store and we do not want to scare visual impaired users away from this great app. Element X has allot of potential making a privacy oriented accessible messenger. I believe that this is the MOST IMPORTANT issue currently.
We are not designing a building without a wheelchair ramp. We are not designing an app that blind people cannot use.

I am on version 1.2.3 and I have the latest IOS.

If their are any questions regarding VoiceOver, contact me at ```@qlp:matrix.org```

**Comment by pannie** [comment]
This is still not fixed?!!

**Comment by sempruijs** [comment]
I've factory rested my iPhone, reinstalled element x and I had the same issue.
Weirdly enough, it is not on any other iOS device I've tried. It is a bit of a mystery but if someone else has the same issue and knows how to reproduce it, please reopen this issue.
